### ISSUE TITLE
Pull in Lender Company when add new loan

### ISSUE THREAD

**Issue by MappingKat** [MappingKat/issues#8]
<img width="593" alt="screenshot 2016-02-07 21 02 09" src="https://cloud.githubusercontent.com/assets/1077436/12878232/1f82b20e-cdde-11e5-8277-bbe18a153530.png">

**Comment by imcmahon** [comment]
We won't be able to pull in the bank lender until it shows up on public records, but it should populate within a couple months. I hid the paragraph for now if we don't have any info at all there and pushed it up.

I'm also surprised that one's not in public record yet. Did Dave put that one in or did you? Wondering if it's an unlisted private loan or something...

**Comment by MappingKat** [comment]
I put that in there. I was going to send you the line he sent, but realized that this is still a public repo.  Do you mind if I move this over to the HomeBot repo?  

I like tracking issues... its easier for me to use then trello. But if you think I should use trello then I can do that as well :)

**Comment by imcmahon** [comment]
Yeah, we should talk through where to keep them. No problem if you want to throw it on the web repo until then though

**Comment by imcmahon** [comment]
https://trello.com/c/44SosPeF
